bin/update-ips.bash

```diff
@@ -628,4 +628,4 @@ if rclone_enabled; then
   sync_rclone '.objectStorage.sync.secondaryUrl' '.networkPolicies.rclone.sync.secondaryUrl'
 fi
 
-exit ${has_diff}
+${dry_run} && exit ${has_diff}
```

Comment:
Can we avoid this `check && conditional` construct?
As it is an exception in behaviour compared to how we otherwise script given that we run with exit on err and pipefail.

And I know I use `check || conditional`, but its logic feels more sane as the final value is still handled under exit on err, while `false && nothing` is an exception to the rule.

Reply:
Yep! https://github.com/elastisys/compliantkubernetes-apps/pull/2448/commits/5cb2572d3e5a584dfaf6105331af780de8625fdb